### DISCUSSION TITLE
Marathon Mod (Automatically choose a random map after passing)

### DISCUSSION THREAD

**Discussion by MilkAndBanana1** [ppy/osu#28965] [Ideas]
A mod that would automatically select a random map in a selected difficulty range would be nice for prolonged practice sessions.

Ideally it would have a small break like when it loads a map in but ideally it goes on until you run out of maps in that selected difficulty range or you fail and it gives you an average of allthe maps that you passed.

**Comment by CloneWith** [comment]
This should be an individual **gamemode or entry** rather than a mod, the same level with Solo, Multiplayer, etc.

Mods are used to change the game's behaviour on *one* beatmap, and in our common sense, one score refers to one beatmap.

**Reply by ahrl9** [reply]
I think this idea kind of matches the dan's other gamemodes or other rhythm games have. BTMC has talked about this topic: https://www.youtube.com/watch?v=z8RnYWS406I, I think this could be a great feature in the future, maybe if it was more official it could also be more relevant for standard players.

You could make some dans badged, like tourneys, so people can see what dan you are. Ofcourse only if the dan is approved and has enough quality.

**Reply by CloneWith** [reply]
seems like an alternative to the original *level* concept
